challenge.rb

```diff
@@ -1,26 +1,27 @@
 
-def capitalize_each_string(input)
+def capitalize_each_string(*animal)
   #implement your solution here
+  animal.each { |a| puts a.capitalize }
```

Comment:
The splat operator allows you to pass multiple parameters that are represented as a single Array in your method. Using the splat operating and passing the method an Array did not work because now your method is wrapping that Array in another Array. Your method was not designed to take an Array of Arrays of Strings, so it tried to run the capitalize method on Array (which doesn't exist) causing the no method error.